homeassistant/components/shelly/climate.py

```diff
@@ -308,10 +313,6 @@ async def async_added_to_hass(self) -> None:
 
```

Comment:
We can remove `should_poll`. It's already implemented by the parent coordinator entity.

Reply:
Done